tests/composite_pk/test_get.py

```diff
@@ -84,30 +84,20 @@ def test_get_or_create_user(self):
 
     def test_lookup_errors(self):
         with self.assertRaisesMessage(
-            ValueError,
-            "The right-hand side of the 'exact' lookup must be a tuple or a list",
+            ValueError, "'primary_key' must be a tuple or a list"
         ):
             Comment.objects.get(pk=1)
```

Comment:
It would be nice if this could be `"'pk' must be a tuple or a list"`.

Reply:
I don't think this is possible unfortunately, `pk` is translated to the field

Reply:
How about this?

> 'exact' lookup of 'primary_key' field must be a tuple or a list